parachain/config.json

```diff
@@ -32,6 +32,7 @@
 		}
 	],
 	"simpleParachains": [],
+	"hrmpChannels": [],
```

Comment:
@vgeddes unrelated q: could something related to this config be what caused weirdness when you implemented XCMP?

Reply:
FWIW this is one of the things I've tried during my journey setting up the local chain